stylesheets/_component.navigation.scss

```diff
@@ -219,6 +219,7 @@ $nav-color-dark:  darken(color(jadu-blue, dark), 5%) !default;
 .nav-primary {
     background-color: $nav-color-light;
     box-shadow: inset 0 0 0 $nav-color-dark;
+    overflow-y: hidden;
```

Comment:
We need all items to be available as a basic requirement, even if that requires scrolling, just not obscured by the scrollbar and definitely no horizontal Scrolling. 

We'd definitely need the third aspect of this ticket addressing before we could break the existing scroll behavior as it would affect customers and create new support requests.